rubin_scheduler/scheduler/features/conditions.py

```diff
@@ -380,12 +379,20 @@ def slewtime(self, value):
 
     @property
     def airmass(self):
+        if self._airmass is None:
```

Comment:
Why not use `@cached_property` instead of the funny dance with `self._airmass`?
If `self.airmass` is your decorated method that computes airmass, and you need to clear the cache, you can delete the property: `del self.airmass`.
The next time you reference `self.airmass` it will get recomputed (and re-cached with the new value).
If you just want to assign it to something that isn't computed by your decorated method, you can just set the cache just by setting the property: 
`self.airmass = 1.5` will set the cache to 1.5.

Reply:
Mostly it makes it analogous with all the other attributes. This way if someone changes the MJD the airmass gets reset to `None` like everything else.